app/Notifications/Admin/AdminCreateUserNotification.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Notifications\Admin;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class AdminCreateUserNotification extends Notification
+{
+    use Queueable;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct($user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+     /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+                    ->subject('User ' . $this->user->email . ' has just registered.')
+                    ->line('A new user ' . $this->user->name . '(' . $this->user->email . ') has just registered.')
```

Comment:
Change the notification text to be:

Welcome to PARCELHUB Warehouse system! 
Your temporary password is {{ password }}
Please click the button below to verify your account, we also recommend changing your password after you verify.